site/components/link/index.js

```diff
@@ -1,19 +1,36 @@
 // @flow
 import React, { PropTypes } from 'react';
+import { StateNavigator } from 'navigation';
 import { NavigationLink } from 'navigation-react';
 
-type LinkProps = { [id: string]: any };
-export default function Link(props: LinkProps) {
-  const { to, ...rest } = props;
+export default class Link extends React.Component {
+  static contextTypes = {
+    stateNavigator: () => {},
```

Comment:
How does this work?

Reply:
Our router ([navigation-react](https://github.com/grahammendick/navigation/tree/master/NavigationReact)) provides the `stateNavigator` in the react context. We can inject this into our component to get access to the currently active route.

Do we want a comment explaining this here?

Reply:
that may be nice - an empty function tends to shout 'unfinished' to me